File: Python/EstruturaDeDecisao/exercicio_01.py
"""
Faça um Programa que peça dois números e imprima o maior deles.
"""

num1 = float(input('Digite o primeiro número: '))
num2 = float(input('Digite o segundo número: '))

if num1 > num2:
    maior = num1
else:
    maior = num2
print(f'O maior número é: {maior}.')

File: Java/EstruturaDeDecisao/Exercicio01.java
// Faça um Programa que peça dois números e imprima o maior deles.
package EstruturaDeDecisao;
import java.util.Scanner;

public class Exercicio01 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o primeiro número: ");
        int numero1 = scanner.nextInt();

        System.out.print("Digite o segundo número: ");
        int numero2 = scanner.nextInt();

        int maiorNumero = numero1 > numero2 ? numero1 : numero2;
        System.out.println("O maior número é: " + maiorNumero);

        scanner.close();
    }
}
